ansible_base/authentication/serializers/authenticator.py

```diff
@@ -103,3 +103,14 @@ def validate(self, data) -> dict:
             return data
         except ImportError as e:
             raise ValidationError({'type': _('Failed to import %(e)s') % {'e': e}})
+
+    def _get_related(self, obj) -> dict[str, str]:
+        related = super()._get_related(obj)
+        from ansible_base.authentication.views.authenticator_users import get_authenticator_user_view
```

Comment:
Is this import recursive if we put it up top?

Reply:
Yea